docs/source/tutorial/basic.rst

```diff
@@ -2,21 +2,564 @@
+
+If we check the contents of the ``node`` variable:
+
+.. code-block:: python
+
+    In [2]: node
+    Out[2]: <Int: uuid: eac48d2b-ae20-438b-aeab-2d02b69eb6a8 (unstored) value: 2>
+
+Quite a bit of information on our freshly created node is returned:
+
+* The data node is of the type ``Int``
+* The node has the *universally unique identifier* (**UUID**) ``eac48d2b-ae20-438b-aeab-2d02b69eb6a8``
+* The node is currently not stored in the database ``(unstored)``
+* The integer value of the node is ``2``
+
+Let's store the node in the database:
+
+.. code-block:: python
+
+    In [3]: node.store()
+    Out[3]: <Int: uuid: eac48d2b-ae20-438b-aeab-2d02b69eb6a8 (pk: 1) value: 2>
+
+As you can see, the data node has now been assigned a *primary key* (**PK**), a number that identifies the node in your database ``(pk: 1)``.
+The difference between the PK and UUID is that the PK is a quick reference to a node that is unique for *one* database only, whereas the UUID is a hexadecimal string that is globally unique between *different* databases.
+
+.. important::
+
+    The PK numbers shown throughout this tutorial assume that you start from a completely empty database.
+    It is likely that the node PK's will be different for your database!
+
+Next, let's leave the IPython shell by typing ``exit()`` and then enter.
+Back in the terminal, use the ``verdi`` command line interface (CLI) to check data node we have just created:
+
+.. code:: bash
+
+    $ verdi node show 1
+
+This prints something like the following:
+
+.. code-block:: bash
+
+    Property     Value
+    -----------  ------------------------------------
+    type         Int
+    pk           1
+    uuid         eac48d2b-ae20-438b-aeab-2d02b69eb6a8
+    label
+    description
+    ctime        2020-05-13 08:58:15.193421+00:00
+    mtime        2020-05-13 08:58:40.976821+00:00
+
+Once again, we can see that the node is of type ``Int``, as PK = 1, and UUID = ``eac48d2b-ae20-438b-aeab-2d02b69eb6a8``. Besides this information, the ``verdi node show`` command also shows the (empty) ``label`` and ``description``, as well as the time the node was created (``ctime``) and last modified (``mtime``).
+
+.. note:: Although AiiDA already provides many data types, you can also :ref:`create your own<how-to:data:plugin>`.
+
+Calculation functions
+=====================
+
+Once your data is stored in the database, it is ready to be used for some computational task.
+For example, let's say you want to multiply two ``Int`` data nodes.
+The following Python function:
+
+.. code-block:: python
+
+    def multiply(x, y):
+        return x * y
+
+will give the desired result when applied to two ``Int`` nodes, but the calculation will not be stored in the provenance.
+However, AiiDA ships with a handy ``decorator`` that you can add to any Python function so its execution will be stored in the provenance.
+Start up the AiiDA IPython shell again using ``verdi shell`` and execute the following code snippet:
```

Comment:
Agreed! I originally left out the prompts because they were also copied with the `sphinx-copybutton`, but @chrisjsewell is working on fixing this with regexes.